client/src/components/organisms/AdminRequestGroupBrowser.tsx

```diff
@@ -0,0 +1,119 @@
+import { bindActionCreators, Dispatch } from "redux"
+import { Col, Row } from "react-bootstrap";
+import { gql, useQuery } from "@apollo/client";
+import React, { FunctionComponent } from "react";
+import { connect } from "react-redux";
+import { loadRequestGroup } from '../../data/actions'
+import RequestGroup from '../../data/types/requestGroup'
+import RequestTypeDropdownList from "./RequestTypeDropdownList";
+import { RootState } from '../../data/reducers'
+import { useParams } from "react-router";
+
+import { Dropdown } from "react-bootstrap"
+
+interface StateProps {
```

Comment:
You don't need to use Redux if this component is the only component where we view the stored requestGroup. Apollo caches all graphql query results and so on reloading a page containing this component the API query should be instant if it has already been loaded recently.

Reply:
@guy-ro  redux is now removed